content/testing/fluent-assertions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: FluentAssertions
+showMetadata: true
+editable: true
+showToc: true
+---
+
+สมมติว่าเรามี Data Structure ดังนี้ คำถามคือเราจะ Assert อย่างไร
+
+```c#
+public class Todo
+{
+    public int id  { get; set; }
+    public string title  { get; set; }
+    public bool completed  { get; set; }
+}
+
+public class TodoList
+{
+
```

Comment:
Very minor, not sure this is from a code formatting in your IDE but please remove an empty line.

Reply:
For me, leaving blank new line will be easy to read

Reply:
FYI, I follow this pattern and it works very well.
https://docs.microsoft.com/en-us/archive/blogs/kirillosenkov/kirills-whitespace-guidelines-for-c